### ISSUE TITLE
Protozoan parasite in CSF wet mount

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11057]
**Preferred term label:**
Protozoan parasite in CSF wet mount

**Synonyms**
Cerebrospinal fluid; Naegleria Acanthamoeba Sappinia and Balamuthia trophozoites included

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Amebic meningoencephalitis; see https://www.ncbi.nlm.nih.gov/books/NBK430754/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001259
- label: Positive protozoan parasite CSF wet mount test
- parents: [Positive CSF infectious agent test (HP:0430123)](https://hpo.jax.org/app/browse/term/HP:0430123)
- pmid: PMID:19635883
- def: Demonstration of protozoan parasites in a cerebrospinal fluid (CSF) specimen by microscopy.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11057
- synonyms:
